lib/core/utils.toit

```diff
@@ -440,11 +440,19 @@ Converts a number between 0 and 15 to an upper case
 to_upper_case_hex c/int -> int:
   return "0123456789ABCDEF"[c]
 
+/**
+Produces a histogram of object types and their memory
+  requirements.  The histogram is sent as a system
+  mirror message, which means it is usually printed on
+  the console.
+*/
 print_objects marker/string="" gc/bool=true:
   if gc:
     before := gc_count
     while gc_count == before: RecognizableFiller_
-  print_histogram_ marker object_histogram_
+  encoded_histogram := object_histogram_
```

Comment:
So the marker is just ignored now? It is pretty useful to understand where a specific histogram came from.